distribution/changelog.txt

```diff
@@ -27,6 +27,7 @@
 - Fix: [#10547] RCT1 parks have too many rides available.
 - Fix: [#10587] Update last action coordinates on correct player.
 - Fix: [#10662] Duck cheat tooltips look odd and do not explain anything.
+- Fix: [#10691] Improved tile inspector delete functionality.
```

Comment:
This would fit better below with the other 'Improved' items, and more descriptive. Something like:
`- Improved: [#10691] Tile elements can now only be removed when there are other tile elements left.`